packages/gatsby-theme-carbon/src/components/AutolinkHeader/AutolinkHeader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import GHSlugger from 'github-slugger';
+import Link from '@carbon/icons-react/lib/link/20';
+import cx from 'classnames';
+
+import { header, anchor } from './AutolinkHeader.module.scss';
```

Comment:
You can destructure the class names from the sass module.

`hypenated-class` would become `hyphenatedClass`

you can also just do `import styles from 'x.module.scss'` and then use styles.anchor etc.